packages/suite/src/views/wallet/transactions/components/TransactionList/index.tsx

```diff
@@ -78,178 +107,76 @@ const TransactionList = ({
-                worker.terminate();
-            };
-        },
-        [
-            isExportRunning,
-            account.descriptor,
-            account.symbol,
-            account.history,
-            addNotification,
-            translationString,
-        ],
-    );
-
-    const exportMenu = useMemo(() => {
-        // Check if the flag is enabled
-        if (!isEnabled('EXPORT_TRANSACTIONS')) {
-            return;
-        }
-
-        // Don't display for Ripple (for now)
-        if (account.networkType === 'ripple') {
-            return;
-        }
-
-        if (isExportRunning) {
-            return <Loader size={18} />;
-        }
-
-        return (
-            <Dropdown
-                alignMenu="right"
-                items={[
-                    {
-                        key: 'export',
-                        options: [
-                            {
-                                key: 'export-csv',
-                                label: <Translation id="TR_EXPORT_AS" values={{ as: 'CSV' }} />,
-                                callback: () => runExport('csv'),
-                            },
-                            {
-                                key: 'export-pdf',
-                                label: <Translation id="TR_EXPORT_AS" values={{ as: 'PDF' }} />,
-                                callback: () => runExport('pdf'),
-                            },
-                            {
-                                key: 'export-json',
-                                label: <Translation id="TR_EXPORT_AS" values={{ as: 'JSON' }} />,
-                                callback: () => runExport('json'),
-                            },
-                        ],
-                    },
-                ]}
-            />
-        );
-    }, [isExportRunning, runExport, account.networkType]);
-
     // if totalPages is 1 do not render pagination
     // if totalPages is undefined check current page and number of txs (e.g. XRP)
     // Edge case: if there is exactly 25 txs, pagination will be displayed
     const isOnLastPage = slicedTransactions.length < SETTINGS.TXS_PER_PAGE;
     const shouldShowRipplePagination = !(currentPage === 1 && isOnLastPage);
-    const showPagination = totalPages ? totalPages > 1 : shouldShowRipplePagination;
+    const isRipple = account.networkType === 'ripple';
+    const showPagination = isRipple ? shouldShowRipplePagination : total > 1;
 
     return (
         <StyledSection
             ref={ref}
             heading={<Translation id="TR_ALL_TRANSACTIONS" />}
-            actions={exportMenu}
+            actions={
+                <Actions
+                    account={account}
+                    search={search}
+                    setSearch={setSearch}
+                    setSelectedPage={setSelectedPage}
+                />
+            }
         >
+            {/* TODO: show this skeleton also while searching in txs */}
```

Comment:
I think that comment was left by @slowbackspace in the design commit. Was this some work intended for me?

Reply:
There wasn't any "loading/search in progress" before (not sure if it is there now) so I just left the comment that the skeleton which is rendered few lines below could be used for that purpose. Sorry I wasn't more clear. Anyway if we don't have any "search in progress" state (since the search is so fast anyway) then you can just delete the comment otherwise adjusting condition should be enough 🙇

Reply:
I think for now it will be alright. We can get some feedback on it and adjust it later.